packages/datagateway-search/src/searchTabs.component.tsx

```diff
@@ -13,29 +11,31 @@ import {
 import {
   CartProps,
   parseSearchToQuery,
-  useDatafileCount,
-  useDatasetCount,
-  useInvestigationCount,
-  useLuceneSearch,
+  SearchResponse,
+  useLuceneSearchInfinite,
   useUpdateQueryParam,
   ViewCartButton,
   ViewsType,
 } from 'datagateway-common';
-import {
-  getFilters,
-  getSorts,
-  storeFilters,
-  storeSort,
-} from './searchPageContainer.component';
-import { connect } from 'react-redux';
-import { useTranslation } from 'react-i18next';
 import { useLocation } from 'react-router-dom';
-import { StateType } from './state/app.types';
+import { useTranslation } from 'react-i18next';
+import { InfiniteData, useIsFetching } from 'react-query';
+import { connect } from 'react-redux';
+import { getFilters, getSorts } from './searchPageContainer.component';
+import type { StateType } from './state/app.types';
 import InvestigationSearchTable from './table/investigationSearchTable.component';
 import InvestigationCardView from './card/investigationSearchCardView.component';
 import DatafileSearchTable from './table/datafileSearchTable.component';
 import DatasetCardView from './card/datasetSearchCardView.component';
 import DatasetSearchTable from './table/datasetSearchTable.component';
+import SearchTabLabel from './searchTabLabel.component';
+
+export interface SearchTableProps {
+  containerHeight: string;
+  hierarchy: string;
+  onTabChange: (currentTab: string) => void;
+  currentTab: string;
+}
```

Comment:
I don't think these are being used anywhere and are a duplicate of `SearchTabsProps` so they can be removed